README.md

```diff
@@ -694,6 +694,12 @@ var signIn = new OktaSignIn(config);
     helpSupportNumber: '(123) 456-7890'
     ```
 
+- **brandName:** The brand name that is displayed in the various descriptions of the Sign In Widget (example: "Signing in to {`brandName`}..."). If no number is provided, no brand name is shown to the user (example: "Signing in...").
```

Comment:
Should this say `If no brandName is provided`?

Reply:
Good catch!